Kms/src/V1/Gapic/KeyManagementServiceGapicClient.php

```diff
@@ -101,32 +103,18 @@
  * ```
  * $keyManagementServiceClient = new KeyManagementServiceClient();
  * try {
- *     $formattedParent = $keyManagementServiceClient->locationName('[PROJECT]', '[LOCATION]');
- *     // Iterate over pages of elements
- *     $pagedResponse = $keyManagementServiceClient->listKeyRings($formattedParent);
- *     foreach ($pagedResponse->iteratePages() as $page) {
- *         foreach ($page as $element) {
- *             // doSomethingWith($element);
- *         }
- *     }
- *
- *
- *     // Alternatively:
- *
- *     // Iterate through all elements
- *     $pagedResponse = $keyManagementServiceClient->listKeyRings($formattedParent);
- *     foreach ($pagedResponse->iterateAllElements() as $element) {
- *         // doSomethingWith($element);
- *     }
+ *     $formattedName = $keyManagementServiceClient->cryptoKeyVersionName('[PROJECT]', '[LOCATION]', '[KEY_RING]', '[CRYPTO_KEY]', '[CRYPTO_KEY_VERSION]');
+ *     $ciphertext = '';
+ *     $response = $keyManagementServiceClient->asymmetricDecrypt($formattedName, $ciphertext);
  * } finally {
  *     $keyManagementServiceClient->close();
  * }
  * ```
  *
- * Many parameters require resource names to be formatted in a particular way. To assist
- * with these names, this class includes a format method for each type of name, and additionally
- * a parseName method to extract the individual identifiers contained within formatted names
- * that are returned by the API.
+ * Many parameters require resource names to be formatted in a particular way. To
+ * assistwith these names, this class includes a format method for each type of
```

Comment:
@miraleung @dwsupplee some wonky newline parsing here. Seems like a bit of a regression.

Reply:
https://github.com/googleapis/gapic-generator-php/pull/302